test/contracts-network/colony-reward-payouts.js

```diff
@@ -276,17 +280,98 @@ contract("Colony Reward Payouts", accounts => {
       const colonyWideReputationKey = makeReputationKey(newColony.address, rootDomainSkill);
       const { key, value, branchMask, siblings } = await client.getReputationProofObject(colonyWideReputationKey);
       colonyWideReputationProof = [key, value, branchMask, siblings];
+      await newColony.moveFundsBetweenPots(1, 0, 0, 1, 0, 100, newToken.address);
 
       await checkErrorRevert(
-        newColony.startNextRewardPayout(otherToken.address, ...colonyWideReputationProof),
+        newColony.startNextRewardPayout(newToken.address, ...colonyWideReputationProof),
         "colony-reward-payout-invalid-total-tokens"
       );
     });
 
-    it("should not be able to create parallel payouts of the same token", async () => {
+    it("should not be able to create parallel payouts of the same token if there are no more tokens to pay out", async () => {
+      await colony.startNextRewardPayout(otherToken.address, ...colonyWideReputationProof);
+      await checkErrorRevert(colony.startNextRewardPayout(otherToken.address, ...colonyWideReputationProof), "colony-reward-payout-no-rewards");
+    });
+
+    it("should be able to create parallel payouts of the same token", async () => {
```

Comment:
After more discussion, we decided that this restriction wouldn't actually provide any guarantees since a malicious user could simply make more colonies to get around the restriction.